doc/rose-rug-files.html

```diff
@@ -444,7 +444,9 @@ <h3 id="rose-config">rose config</h3>
     <div class="slide">
       <h3 class="alwayshidden">rose config --file</h3>
 
-      <p>rose config also works for the other Rose formats. For example:</p>
+      <p>rose config also works for other Rose configuration files (and any
```

Comment:
Could you pop this in ```<kbd>``` tags like the other commands (I know it wasn't originally but while you're in here we may as well change it)